src/test/deserialization_tests.cpp

```diff
@@ -222,8 +230,8 @@ TEST_P(DeserializeTFTensorProto, ShouldReturnValidBlob) {
     Precision testedPrecision = GetParam();
     SetUpTensorProto(fromInferenceEnginePrecision(testedPrecision));
     tensorMap[tensorName]->setPrecision(testedPrecision);
-    // InferenceEngine::Blob::Ptr blobPtr = deserializeTensorProto<ConcreteBlobGenerator>(tensorProto, tensorMap[tensorName]);
-    InferenceEngine::Blob::Ptr blobPtr = deserializeTensorProto<ConcreteTensorProtoDeserializator>(tensorProto, tensorMap[tensorName]);
+    bool isPipeline = false;
```

Comment:
If all tests in this suite are non-pipeline, maybe move this variable to fixture?